src/test/resources/application.conf

```diff
@@ -34,7 +34,7 @@ h2mem1 {
   db {
     connectionPool = disabled
     driver = "org.h2.Driver"
-    url = "jdbc:h2:mem:test1;DATABASE_TO_UPPER=FALSE"
+    url = "jdbc:h2:mem:test1;DATABASE_TO_UPPER=FALSE;MODE=MySQL"
```

Comment:
Doesn't fix the tests I had to ignore, but we should use this while we still use H2